### PR TITLE
Dockerfile Update - Julia

### PR DESCRIPTION
**Pull request by calvinp0**
RMG-Py environment has now been updated to restrict the Julia version. Since this has been done, we do not need to patch the environment file ourselves.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -118,7 +116,9 @@ RUN echo "alias rmge='micromamba activate rmg_env'" >> ~/.bashrc \
     && echo "alias rmgdb='cd \$rmgdb_path'" >> ~/.bashrc \
     && echo "alias arcode='cd /home/rmguser/Code/ARC'" >> ~/.bashrc \
     && echo "alias conda='micromamba'" >> ~/.bashrc \
-    && echo "alias mamba='micromamba'" >> ~/.bashrc
+    && echo "alias mamba='micromamba'" >> ~/.bashrc \
```
Comment (MichalKesl) [inline Dockerfile:119]: shouldn't this be  >> ~/.bashrc (without \ at the end)?

Comment (calvinp0) [reply Dockerfile:119]: This is just how Dockerfile needs to be written if you are having multiple RUN statements put together

Comment (calvinp0) [reply Dockerfile:119]: `\` will always be followed by `&&` to denote that there is another statement that will occur in that RUN layer (statement)

**Review by MichalKesl — APPROVED** [review]
Looks great

**Pull request merged**